templates/settings/store-form.php

```diff
@@ -182,6 +188,53 @@
             </div>
         </div>
 
+        <?php
+        $sections = array_filter( $products_block_appearance, function( $value ) {
+            return empty( $value );
+        } );
+
+        if ( count( $sections ) > 0 ) {
+            ?>
+            <div class="dokan-form-group">
+            <label class="dokan-w3 dokan-control-label"><?php esc_html_e( 'Store page product sections', 'dokan-lite' ); ?></label>
```

Comment:
this `<label>` should have proper alignment according to it's parent `<div>`.

and same for others.